src/GovUk.Education.ExploreEducationStatistics.Public.Data.Processor/ProcessorHostBuilder.cs

```diff
@@ -0,0 +1,93 @@
+using Azure.Core;
+using Azure.Identity;
+using GovUk.Education.ExploreEducationStatistics.Common.Database;
+using GovUk.Education.ExploreEducationStatistics.Common.Extensions;
+using GovUk.Education.ExploreEducationStatistics.Content.Model.Database;
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Model.Database;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Npgsql;
+
+namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Processor;
+
+public static class ProcessorHostBuilder
+{
+    public static IHostBuilder ConfigureProcessorHostBuilder(this IHostBuilder hostBuilder)
+    {
+        return hostBuilder
+            .ConfigureAppConfiguration(builder =>
+            {
+                builder
+                    .AddJsonFile("appsettings.json", optional: true, reloadOnChange: false)
+                    .AddJsonFile("appsettings.Local.json", optional: true, reloadOnChange: false)
+                    .AddEnvironmentVariables();
+            })
+            .ConfigureLogging(logging =>
+            {
+                // TODO Why can't this be controlled through application settings?
```

Comment:
I've already spent quite a lot of time trying to resolve this, probably a few hours as part of this ticket and just spent another 45 mins now!

I'm tempted to just leave this TODO here rather than spend any more time on it or create a separate ticket for it. There's already a ticket related to this which is EES-5013.

Reply:
Maybe just update to say `TODO EES-5013` instead so we know it's covered and known about?